code/game/machinery/computer/message.dm

```diff
@@ -69,11 +69,15 @@
 	..()
 
 /obj/machinery/computer/message_monitor/Initialize()
+	..()
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/machinery/computer/message_monitor/LateInitialize()
+	. = ..()
```

Comment:
LateInitialize doesn't actually do initialization in the same sense as Initialize so its name is a little misleading; "PostInitialize" would probably be better semantically. Regardless, there's no need to return a hint from super here.